crates/symbolicator/src/service.rs

```diff
@@ -104,8 +106,9 @@ pub enum SymbolicationResponse {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[serde(untagged)]
 pub enum CompletedResponse {
-    NativeSymbolication(CompletedSymbolicationResponse),
-    JsSymbolication(CompletedJsSymbolicationResponse),
+    Native(CompletedSymbolicationResponse),
+    Js(CompletedJsSymbolicationResponse),
+    Jvm(CompletedJvmSymbolicationResponse),
```

Comment:
Can we add some comments to each enum value?

Reply:
I added some docstrings. They are a bit boilerplate because there's not that much to say about these variants, but I'll never say no to a request for documentation :)